src/devtools/makefile_base_posix.mak

```diff
@@ -575,7 +577,7 @@ $(GAMEOUTPUTFILE): $(OUTPUTFILE)
 		echo "----" $(QUIET_ECHO_POSTFIX);\
 		$(P4_EDIT_START) $(IMPORTLIBRARY) $(P4_EDIT_END) && \
 		mkdir -p `dirname $(IMPORTLIBRARY)` > /dev/null && \
-		cp -v $(OUTPUTFILE) $(IMPORTLIBRARY); \
+		cp $(OUTPUTFILE) $(IMPORTLIBRARY); \
```

Comment:
Why was -v removed?

Reply:
Makes build output quieter

Reply:
I've reverted this, since I didn't fully change every instance of `cp`, no reason to have these be special